### DISCUSSION TITLE
Data-importer: 1.0.2 - Nordigen - Invalid currency code NOK

### DISCUSSION THREAD

**Discussion by Alpoy** [firefly-iii/firefly-iii#6937] [Firefly III] [answered]
Hi! 

I tried the new data-importer 1.0.2 with nordigen, but all transaction lines are ignored giving this error when importing (after mapping):
Line #0: transactions.0.currency_code: Den valgte transactions.0.currency_code er ikke gyldig. (original value: "NOK")
...

Firefly (5.7.18) is configured to only use "NOK".

**Comment by JC5** [comment]
> Firefly (5.7.18) is configured to only use "NOK".

What do you mean exactly?

**Reply by Alpoy** [reply]
I just meant that I've enabled "NOK" as default currency in Firefly (Euro is the only active _alternative_ currency but not the default).

**Reply by JC5** [reply]
Alright, what's your config like? I can't replicate this.

**Reply by Alpoy** [reply]
Import config: 
> {
    "version": 3,
    "source": "fidi-1.0.2",
    "created_at": "2023-01-29T10:02:06+01:00",
    "date": "",
    "default_account": 1,
    "delimiter": "comma",
    "headers": false,
    "rules": true,
    "skip_form": false,
    "add_import_tag": true,
    "roles": [],
    "do_mapping": [],
    "mapping": [],
    "duplicate_detection_method": "none",
    "ignore_duplicate_lines": false,
    "unique_column_index": 0,
    "unique_column_type": "external-id",
    "flow": "nordigen",
    "identifier": "0",
    "connection": "0",
    "ignore_spectre_categories": false,
    "map_all_data": true,
    "accounts": {
        "<encrypted-key>": 1,
        "<encrypted-key>": 4,
        "<encrypted-key> 5
    },
    "date_range": "range",
    "date_range_number": 30,
    "date_range_unit": "d",
    "date_not_before": "2023-01-01",
    "date_not_after": "2023-01-29",
    "nordigen_country": "NO",
    "nordigen_bank": "SPAREBANK_NORD_NORGE_SNOWNO22",
    "nordigen_requisitions": {
        "<encrypted-key>": "<encrypted-key>"
    },
    "nordigen_max_days": "29",
    "conversion": false,
    "ignore_duplicate_transactions": false
}

**Comment by JC5** [comment]
Mmmm, not very conclusive. Alright, could you share the logs of a run with me? Can be at james@firefly-iii.org

**Comment by Alpoy** [comment, marked as answer]
Figured this one out, it was a (PEBKAC) user error...when I added my currency (NOK/Norwegian Kroner) I've use a numeric code for "CODE" instead of the actual currency code (NOK). Sorry for wasting your time @JC5 with this.

**Reply by JC5** [reply]
Not a problem! Glad to hear you found the issue!
